go-controller/pkg/metrics/ovn.go

```diff
@@ -275,7 +274,7 @@ func setOvnControllerConfigurationMetrics() (err error) {
 			metricOpenFlowProbeInterval.Set(metricValue)
 		case "ovn-remote-probe-interval":
 			metricValue := parseMetricToFloat(MetricOvnSubsystemController, "ovn-remote-probe-interval", fieldValue)
-			metricRemoteProbeInterval.Set(metricValue)
+			metricRemoteProbeInterval.Set(metricValue / 1000)
```

Comment:
I think this should be already in seconds, since `ovn-openflow-probe-interval=180` on my cluster and https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/node/node.go#L168-L169 is set using config option https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/config/config.go#L178-L180 that says it should be in seconds